velox/exec/tests/SpillerTest.cpp

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/exec/Spiller.h"
+#include <folly/executors/IOThreadPoolExecutor.h>
+#include "velox/exec/tests/utils/RowContainerTestBase.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::exec;
+
+class SpillerTest : public exec::test::RowContainerTestBase {
+ protected:
+  void testSpill(int32_t spillPct, bool makeError = false) {
```

Comment:
This method is super long. Would you split it into multiple shorter methods for modularity and readability?